src/lib.rs

```diff
@@ -774,6 +776,13 @@ impl Builder {
         self
     }
 
+    /// Set whether or not to time bindgen phases, and print
+    /// information to stderr.
+    pub fn time_phases(mut self) -> Self {
```

Comment:
For consistency with our other on/off features and options, this should have a `doit: bool` parameter, and set `self.options.time_phases = doit;`